napps/kytos/of_flow_manager/README.rst

```diff
@@ -32,20 +32,21 @@ Protocol Operation
 
 The endpoints implemented in this napp are presented in the table below.
 
-+----------------------------------------+----------------------------------+------------+
-| Endpoint                               | Description                      | Method     |
-+========================================+==================================+============+
-| ``/flow-manager/dpid/flows``           | Retrieve flows from a specific   | ``GET``    |
-|                                        | Switch                           |            |
-+----------------------------------------+----------------------------------+------------+
-| ``/flow-manager/flows``                | Retrieve all flows               | ``GET``    |
-+----------------------------------------+----------------------------------+------------+
-| ``/flow-manager/dpid/flows-a``         | Add a flow in a specific Switch  | ``POST``   |
-|                                        |                                  |            |
-+----------------------------------------+----------------------------------+------------+
-| ``/flow-manager/dpid/flow_id/flows-d`` | Delete a flow from a specific    | ``DELETE`` |
-|                                        | Switch                           |            |
-+----------------------------------------+----------------------------------+------------+
++------------------------------------------+----------------------------------+------------+
+| Endpoint                                 | Description                      | Method     |
++==========================================+==================================+============+
+| ``/flow-manager/flows[/dpid]``           | Retrieve installed flows from    | ``GET``    |
```

Comment:
@diraol yes, we can have a list of all flows, no problem.

I think that the syntax should be something like: this

`/flow-manager/flows/[<dpid>]`

DELETE for all is good. No problem. We don't need to force the user to send a DELETE per switch or per flow. 

About the POST for all.. since that we have to handle different jsons (with and without a switch), yes, maybe we should disable POST all for now.

Remember this is beta and experimental. On the future we have to implement authentication and authorization.